app/assets/stylesheets/layout/page_specific/_money_navigator_questionnaire.scss

```diff
@@ -132,6 +132,10 @@
 			padding: 0 0 0 $baseline-unit * 4;
 		}
 	}
+
+	.question__response__hidden {
```

Comment:
Could we rename this to `question__response--hidden` which more closely follows our BEM convention?